controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -326,6 +329,26 @@ func (r *NodeNetworkConfigurationPolicyReconciler) SetupWithManager(mgr ctrl.Man
 	return nil
 }
 
+func (r *NodeNetworkConfigurationPolicyReconciler) forceNNSRefresh(name string) {
+	log := r.Log.WithName("forceNNSRefresh").WithValues("node", name)
+	log.Info("forcing NodeNetworkState refresh after NNCP applied")
+	nns := &nmstatev1beta1.NodeNetworkState{}
+	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: name}, nns)
+	if err != nil {
+		log.Error(err, "failed retrieving NodeNetworkState to force refresh")
+		return
+	}
+	if nns.Labels == nil {
+		nns.Labels = map[string]string{}
+	}
+	nns.Labels[state.ForceRefreshLabel] = fmt.Sprintf("%d", time.Now().UnixNano())
+
+	err = r.Client.Update(context.Background(), nns)
```

Comment:
Shouldn't we care about conflicts? Or does a conflict mean that the nns was refreshed and thus no need to do so?

Reply:
Kind of, since in a minute is going to be refreshed I didn't want to add conflicts here, but I can try.

Reply:
Undrestood, it's probably not necessary then